src/Components/ManageTeam/EmployeesTable.tsx

```diff
@@ -0,0 +1,165 @@
+import AddIcon from '@mui/icons-material/Add';
+import DeleteIcon from '@mui/icons-material/Delete';
+import EditIcon from '@mui/icons-material/Edit';
+import {
+  Button,
+  IconButton,
+  Paper,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Toolbar,
+  Tooltip,
+  Typography,
+} from '@mui/material';
+import { useState } from 'react';
+
+import { ConfirmDialog } from './ConfirmDialog';
+import { EmployeeForm } from './EmployeeForm';
+import { Notify } from './Notify';
+import { PopUpEmployeeForm } from './PopUpEmployeeForm';
+
+function createData(id: number, login: string, password: string, role: string) {
```

Comment:
Again, `role` could be a union. This union could be placed in one central place e.g. a separate file and imported here.